src/main/java/org/programmers/voucher/io/ConsoleInput.java

```diff
@@ -0,0 +1,38 @@
+package org.programmers.voucher.io;
+
+import org.programmers.voucher.domain.VoucherType;
+import org.programmers.voucher.util.Command;
+import org.programmers.voucher.util.IllegalCommandException;
+import org.programmers.voucher.util.IllegalVoucherTypeException;
+
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.Scanner;
+
+public class ConsoleInput implements Input{
+    private final Scanner sc = new Scanner(System.in);
+
+    @Override
+    public Command inputCommand() throws IllegalCommandException {
+        String input = sc.nextLine().strip();
+        Optional<Command> inputCommand = Arrays.stream(Command.values())
+                .filter(value -> value.toString().equals(input))
+                .findAny();
+        if (inputCommand.isEmpty()){
+            throw new IllegalCommandException();
+        }
+        return inputCommand.get();
```

Comment:
```suggestion
        return Arrays.stream(Command.values())
		        .filter(value -> value.toString().equals(input))
		        .findAny()
		        .orElseThrow(IllegalCommandException::new);
```

이렇게 stream을 사용하는 김에 한번에 처리할 수 있지 않을 까요?